### PR TITLE
DD-5: implement tests for twitter mentions

### PR DESCRIPTION
**Pull request by treeoverflow**
# Story

[See more details for the story](https://easyband.youtrack.cloud/issue/DD-5/Setup-environment-for-tests)

# Description

Needs to setup environment for tests; implement tests for twitter mentions.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

By unit tests.

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by ktsx — APPROVED** [review]
LGTM, only minor things left.

File: src/test/twitter-mentions-count.test.js [hunk 1]
```diff
@@ -0,0 +1,966 @@
...
+const endpointURL = 'https://api.twitter.com/2/tweets/counts/recent';
+const token = process.env.TWITTER_API_KEY;
+
+const mockedMentionsByKeyword = {
+  data: [
+    {
+      end: '2022-12-02T21:00:00.000Z',
+      start: '2022-12-02T20:24:04.000Z',
+      tweet_count: 48,
+    },
```
[the remark this answers is not in the document]
Comment (treeoverflow) [reply src/test/twitter-mentions-count.test.js:13]: Done!

File: jest.config.js [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+  // testLocationInResults: false,
+
+  // The glob patterns Jest uses to detect test files
+  // testMatch: [
+  //   "**/__tests__/**/*.[jt]s?(x)",
+  //   "**/?(*.)+(spec|test).[tj]s?(x)"
+  // ],
+
+  // An array of regexp pattern strings that are matched against all test paths, matched tests are skipped
+  // testPathIgnorePatterns: [
```
[the remark this answers is not in the document]
Comment (treeoverflow) [reply jest.config.js:162]: Done!

File: .eslintrc.json [hunk 3]
```diff
@@ -6,11 +6,21 @@
     },
     "extends": "airbnb-base",
     "overrides": [
+        {
+            "files": [
+                "**/*.test.js",
+                "**/*.test.jsx"
```
[the remark this answers is not in the document]
Comment (treeoverflow) [reply .eslintrc.json:12]: Done!

**Pull request merged**